src/ledger/LedgerManager.h

```diff
@@ -147,8 +147,8 @@ class LedgerManager
     virtual void startNewLedger() = 0;
 
     // loads the last ledger information from the database
-    // if handler is set, also loads bucket information and invokes handler.
-    virtual void loadLastKnownLedger(std::function<void()> handler) = 0;
+    virtual void loadLastKnownLedger(bool startServices,
```

Comment:
Nit: It would be helpful to add what the parameters mean to this comment. `startServices` is pretty clear, `ledgerStateReady` is not obvious unless you read the implementation. Maybe slightly rewording to `isLedgerStateReady` would help as well.